index.js

```diff
@@ -442,11 +444,15 @@ module.exports = {
             ssb.box2.listGroupIds({ excluded: true }),
           ]),
           pull.flatten(),
+          pull.asyncMap((groupId, cb) => ssb.box2.getGroupInfo(groupId, cb)),
           pull.drain(
-            (groupId) => myGroups.add(groupId),
+            (groupInfo) =>
+              groupInfo.readKeys
+                .map((readKey) => readKey.key.toString('base64'))
+                .forEach((readKeyString) => myReadKeys.add(readKeyString)),
```

Comment:
This works, but you could do it will pull-streams tidily too
```js
pull.flatten()
pull.map(readKey => readKey.gey.toString('base64')),
pull.drain(
  myReadKeys.add,
  err => ...
)
```

Reply:
just seems slightly more complex?